docs/docs/graphql.md

```diff
@@ -1087,3 +1201,11 @@ This might be one of our most frequently asked questions of all time. Here's [To
 
 <!-- TODO -->
 <!-- This https://community.redwoodjs.com/t/how-to-add-resolvetype-resolver-for-interfaces/432/7 -->
+
+## Futher Reading
+
+Eager to learn more about GraphQL? Check out some of the resources below:
+
+- [GraphQL.wtf](https://graphql.wtf) covers most aspects of GraphQL and publishes one short video a week 
+- [howtographql.com] is aimed at people looking to learn GraphQL in a practical way, through a set of complete full-stack tutorials
```

Comment:
it's not so up to date tutorial...
Should maybe change that to the Yoga tutorial?
It's more up to date and also is the thing that Redwood actually use under the hood?

Reply:
@Urigo Fixed!